pkg/csi/service/node.go

```diff
@@ -180,28 +180,13 @@ func (driver *vsphereCSIDriver) NodePublishVolume(
 
 	// Check if this is a MountVolume or BlockVolume.
 	if !common.IsFileVolumeRequest(ctx, caps) {
-		params.DiskID, err = driver.osUtils.GetDiskID(req.GetPublishContext(), log)
+		var dev *osutils.Device
+		err = driver.osUtils.VerifyVolumeAttachedFillParams(ctx, req.GetPublishContext(), &params, &dev)
```

Comment:
Is this AttachedFillParams or File Parameters?

Reply:
done